### PR TITLE
move mmseg templates

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

File: tests/ote_cli/external/mmsegmentation/test_ote_cli_tools_segmentation.py [hunk 1]
```diff
@@ -65,6 +65,7 @@
 ote_dir = os.getcwd()
 
 templates = Registry('external').filter(task_type='SEGMENTATION').templates
+templates = [templates[0]]
```
Comment (evgeny-izutov) [inline tests/ote_cli/external/mmsegmentation/test_ote_cli_tools_segmentation.py]: only one?

Comment (Ilya-Krylov) [reply tests/ote_cli/external/mmsegmentation/test_ote_cli_tools_segmentation.py]: oh, thanks!

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
ote-test/1423/ 🕐

**Comment by Ilya-Krylov** [comment]
ote-test/1426/ 🕐

**Comment by Ilya-Krylov** [comment]
ote-test/1428/ 🕐

**Comment by Ilya-Krylov** [comment]
ote-test/1429/

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

File: external/mmsegmentation/segmentation_tasks/sample/sample.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
+# Copyright (C) 2021 Intel Corporation
```
Comment (Ilya-Krylov) [inline external/mmsegmentation/segmentation_tasks/sample/sample.py:1]: the same question here https://github.com/openvinotoolkit/training_extensions/pull/972#discussion_r828151909

@evgeny-izutov ?

**Comment by Ilya-Krylov** [comment]
https://ci-ote.iotg.sclab.intel.com/job/ote/job/ote-test/1437/

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Pull request merged**